Clinical-Genomics/scout
# Do not inactivate case whenever the assignee is removed from database using the cli

fix #2039.

Current situation is that whenever a user is assignee of one or more cases, if the user is removed using the command line, those cases become inactive. This fix will keep those cases in their status, whichever that is.


**How to test --in a local instance of Scout**:
---> From master branch:
1. Create a dummy user to assign the case to:
`scout --demo load user -u "Dummy User" -m dummyuser@dummysite.se -i cust000`
1. Log in into scout with the dummy users's email
1. Enter the case page and then one of its variants; you should get a notification that the case is active.
1. Assign yourself to the case and log out.
1. Remove dummy user from the command line:
`scout --demo delete user -m dummyuser@dummysite.se`
1. Log in as Clark Kent and note that the case is inactivated

--> Repeat from this branch:
1. In the last step you should see that the case is still active

Try to assign whichever state to the case (not inactive of course) to the case before removing the user and the case should keep this status.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
